content/2016.12.give.a.new.chance.to.nightly.rst

```diff
@@ -0,0 +1,223 @@
+10 reasons you should give another chance to Firefox Nightly
+############################################################
+
+:slug: give-firefox-nightly-a-new-chance
+:date: 2016-12-22
+:lang: en
+:url: give-firefox-nightly-a-new-chance
+:summary:
+    Help us make the best Mozilla Firefox of all the times by mid 2017!
+:status: draft
+
+What is Nightly?
+================
+
+Firefox Nightly build also known as Nightly is a version of Firefox
+built from the source code everyday at night.
+
+This means that Firefox updates itself everyday!
+
+
+Why aren't you using Nightly today?
+===================================
+
+Anyone wants a stable Firefox and you don't have time for a crappy
+browser experience.
+
+Also, it was not always the case, Firefox Nightly is now much more
+stable than ever. The number of crashes has been schrinked down and it
+is getting lower really fast everyday.
+
+Also most of the bugs that you might have will be fixed the day after.
+
+You can always go in the `Help > About Firefox` menu to make sure you
+have the last update!
+
+
+Why you should actually give Firefox Nightly a try
+==================================================
+
+By installing Firefox Nightly you will:
+
+#. Have new Mozilla Firefox features 18 weeks before
+#. Have a much faster Firefox
+#. Make a difference
+#. Help reduce the number of crashes for Mozilla Firefox users
+#. Make sure Mozilla Firefox source code is in a workable state
```

Comment:
Nightly is a compiled version of the source code, by using Nightly you make sure the source code of Firefox is still working.